fl4health/clients/moon_client.py

```diff
@@ -1,10 +1,14 @@
+from logging import INFO
```

Comment:
For Moon, perhaps we can do something similar to Ditto, where instead of changing the behavior of the default Moon implementation, we can inherit from Moon and strap on the additional MkMMD behavior?

Reply:
Alternatively, since we're replacing the contrastive loss with the MkMMD loss here, perhaps we just create a separate MOON class for MkMMD.

Reply:
I think strapping is better as we are using some functionality of moon such as saving global moon and etc. I will just put up an error message about contrastive loss weight being zero and not being computed.

Reply:
Okay,  I'll take a look at the implementation.